_data/topnav.yml

```diff
@@ -64,15 +64,13 @@ topnav:
       url: /docs/
     - title: Blog
       url: /blog/
-    - title: Help
-      url: /help/
 topnav_dropdowns:
 - title: Topnav dropdowns
   folders:
-    - title: Contributing
+    - title: Contributing/Help
       folderitems:
-        - title: Get Involved
-          url: /get_involved/
+        - title: Get Involved/Help
```

Comment:
Is it "Help/Get Involved", "Contributing/Help", or Get Involved/Help"? Sidebar, topnav, and page content should prob all agree.

Reply:
"Help/Get Involved"